src/flint/FlintTypes.jl

```diff
@@ -2352,8 +2353,8 @@ end
 A finite field. The constructor automatically determines a fast implementation.
 """
 @attributes mutable struct FqField <: FinField
-   # fq_default_ctx_struct is 200 bytes on 64 bit machine
-   opaque::NTuple{200, Int8}
+   # fq_default_ctx_struct is 208 bytes on 64 bit machine
+   opaque::NTuple{208, Int8}
```

Comment:
@fieker Does that make sense? The corresponding C struct got another `fmpz_mod_ctx_struct *`.